.mci.yml

```diff
@@ -877,15 +877,17 @@ buildvariants:
             - ubuntu1804-build
           - name: uninstall_check
 
+    # TODO CXX-2260 Upgrade std::experimental tasks to use Ubuntu 18.04 and mongodb_latest
+    # after bug fixes with g++ 5.5
```

Comment:
I think that g++5.5 actually worked and it was something around g++7.3 and newer that gave us problems.

Reply:
Ah good point. Made the comment refer generically to `newer g++ versions`.